.gitignore

```diff
@@ -2,7 +2,7 @@
 
 *.exe
 src/bitcoin
-src/bitcoind
+src/bitcoinabc
```

Comment:
@jasonbcox , @tkembo 

src/bitcoind is not referring to a directory.

Might be a technicality here, but it suggests you don't understand the structure of the project (in this case the gitignore) well enough yet.

src/bitcoind in the gitignore refers the compiled bitcoin daemon.

Others have suggested 'bitcoinabcd' as a replacement, but if you're going to propose such a renaming, your PR needs to do it consistently over all project files.
Right now, you still need to meet the listed objectives in your PR description.

To save you time, if you want to do this renaming work seriously, you should register yourself an account on https://reviews.bitcoinabc.org and join the BTCforks slack via self-invite at https://btcforks.signup.team/  so that myself or some other dev can guide you on how to contribute good code.